src/main/webapp/database-tables.js

```diff
@@ -68,6 +79,67 @@ function login() {
   });
 }
 
+//TODO: Pass another paramater to combine data conversion functions 
+function dataConversionMillis(column,tableData){
+    tableData.columnSchemas[column].schemaType = "TIMESTAMP";
+    for (var i =0; i<tableData.dataTable.length; i++) {
+      var time = parseInt(tableData.dataTable[i][column]);
+      var date = new Date(time); 
+      date.setHours(date.getHours() - 2); //Convert from default Central Time to PST
+      tableData.dataTable[i][column] = date.toISOString();
+    } 
+    return tableData;
+}
+
+function dataConversionGenreEnum(column,tableData){
+    tableData.columnSchemas[column].schemaType = "STRING";
+    for (var i =0; i<tableData.dataTable.length; i++) {
+      var number = parseInt(tableData.dataTable[i][column]);
+      switch(number) {
+        case 1:
+            tableData.dataTable[i][column] = "Rock";
+            break;
+        case 2:
+            tableData.dataTable[i][column] = "Jazz";
+            break;
+        case 3:
+            tableData.dataTable[i][column] = "Classical";
+            break;
+        case 4:
+            tableData.dataTable[i][column] = "Pop";
+            break;
+        default:
+           tableData.dataTable[i][column] = "Unspecified";
+      }
+    }
+    return tableData;
+}
+
+function dataConversionProto(column,tableData) {
+    //TODO: work on clicking the { to open and close
+    for (var i = 0; i<tableData.dataTable.length; i++) {
+      var json = tableData.dataTable[i][column];
+      var newString = "";
+      var tab = Boolean(false);
+
+      for (var j = 0; j<json.length; j++) {
+          if (json[j] == ',' ) {
+              newString += (Boolean(tab) ? (json[j] + "\n"+"\t") : (json[j] + "\n"));
```

Comment:
Could you make the "\n" + "\t" into "\n\t"?

Reply:
Done!